app/client/api/authApi.js

```diff
@@ -76,8 +74,7 @@ export const activateAPI = (activation_token) => {
       })
     })
     .then(response => {
-      // special use case for TFA
-      return response.json();
+      return handleResponse(response)
```

Comment:
@enjeyw I'm pretty sure this needs to be `response.json()` to be correctly handled in the saga